packages-exp/auth-types-exp/index.d.ts

```diff
@@ -411,3 +411,18 @@ export interface UserMetadata {
   readonly creationTime?: string;
   readonly lastSignInTime?: string;
 }
+
+/**
+ * Additional user information.
+ */
+export interface AdditionalUserInfo {
+  readonly isNewUser: boolean;
+  readonly profile?: UserProfile;
+  readonly providerId: ProviderId | null;
+  readonly username: string | null;
```

Comment:
looks like username should be defined as `username?: string | null` we should go through later and determine how we consolidate these optional | null properties

Reply:
Good catch!